.eslintrc.yml

```diff
@@ -12,4 +12,5 @@ plugins:
 rules:
   chai-expect/no-inner-compare: warn
   chai-expect/missing-assertion: warn
-  chai-expect/terminating-properties: warn
\ No newline at end of file
+  chai-expect/terminating-properties: warn
+  promise/no-native: off
```

Comment:
why?

Reply:
The promise library is injected using `options`. Do you think it should just be `bluebird`, or maybe default to `bluebird`?